generators/generator-bot-template-generator/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "generator-bot-template-generator",
+  "version": "0.0.1",
```

Comment:
Let's snap to 1.0.0 and use @ryanlengel's pipeline changes for the purposes of adding version suffix etc until we're comfortable to do a GA release.

Reply:
Changed version # and spoke with Ryan about pushing yaml file in a separate PR